vega_sim/service.py

```diff
@@ -512,6 +531,18 @@ def create_asset(
         self.wait_fn(60)
         self.wait_for_thread_catchup()
 
+        asset_id = self.find_asset_id(
+            symbol=symbol, enabled=True, raise_on_missing=True
+        )
+        for _ in range(20):
```

Comment:
Should we not just raise the max faucet amount rather than doing it 20 times, or is it so fast as not to matter? :)

Reply:
Yeah fair point this was a quick fix that avoiding updating every scenario but agree it would be cleaner, will update it now 👍